testhelpers/clock.go

```diff
@@ -1,34 +1,20 @@
 package testhelpers
 
 import (
+	"github.com/jonboulle/clockwork"
 	"time"
 )
 
-// FakeSystemClock returns a mocked clock implementation that may be manually
-// set for testing things related to time.
+// FakeSystemClock wrapps the jonboulle/clockwork.FakeClock interface.
+// FakeSystemClock provides an interface for a clock which can be
+// manually advanced through time.
 type FakeSystemClock struct {
-	now time.Time
+	clockwork.FakeClock
```

Comment:
I don't think the wrapper adds much value. Test code can just call `clockwork.NewFakeClockAt` directly.

Reply:
In the event we decide to replace the `clockwork` dep with our own implementation I'd like to avoid re-writing imports in tests that use it. Any objections to:
```go
type FakeClock interface {
        clockwork.FakeClock
}

// NewFakeClock returns a FakeClock initialised at the given time.Time.
func NewFakeClock(n time.Time) FakeClock {
        return clockwork.NewFakeClockAt(n)
}
```